src/NuGet.Clients/NuGet.CommandLine/Commands/InitCommand.cs

```diff
@@ -30,7 +31,7 @@ public class InitCommand : Command
             // If the Destination Feed Folder does not exist, it will be created.
             OfflineFeedUtility.ThrowIfInvalid(destination);
 
-            var packagePaths = GetPackageFilePaths(source, "*" + ProjectManagement.Constants.PackageExtension);
+            var packagePaths = GetPackageFilePaths(source, "*" + PackagingCoreConstants.NupkgExtension);
```

Comment:
Packaging core is just id + version + dependencies, it doesn't understand the nuget part of things. Packaging adds that in with logic for the different folders.